src/controllers/handlers/scene-admin-handlers/list-scene-admins-handler.ts

```diff
@@ -37,12 +40,8 @@ export async function listSceneAdminsHandler(
     }
   }
 
```

Comment:
I think we can remove the previous logic of returning the status 404 when the place is not found since the place component is throwing the error when this happens.

Reply:
good catch